Donation-platform/src/main/java/com/google/FindTrendingCharities.java

```diff
@@ -97,7 +101,7 @@ private double calcCharityNavRating(Charity charity) {
     }
 
```

Comment:
where are we using this parameter? or is it for now hardcoding? Add a comment in that case so we remember.

Reply:
added comment